src/Common/src/System/Data/Common/AdapterUtil.cs

```diff
@@ -486,11 +486,9 @@ internal static Exception InvalidSeekOrigin(string parameterName)
         }
 
         internal static readonly bool IsWindowsNT = (PlatformID.Win32NT == Environment.OSVersion.Platform);
```

Comment:
Can this line `IsWindowsNT` be removed as well?

Reply:
Yes, certainly :)